docs/reference/central-configuration.md

```diff
@@ -0,0 +1,76 @@
+---
+navigation_title: Central configuration
+description: Reference documentation for the central configuration of EDOT SDKs.
+applies_to:
+  deployment:
+      ess: preview 9.1
+  stack: preview 9.1
+products:
+  - id: cloud
+  - id: observability
+  - id: kibana
+---
+
+# Central configuration for EDOT SDKs
+
+Manage {{edot}} (EDOT) SDKs through the APM central configuration feature in the Applications UI. Changes are automatically propagated to the deployed [EDOT SDKs](./edot-sdks/index.md). Refer to [APM central configuration](docs-content://solutions/observability/apm/apm-agent-central-configuration.md) for more information.
+
+## Prerequisites
+
+To use central configuration for EDOT SDKs, you need:
+
+* An Elastic self-managed or {{ecloud}} deployment, version 9.1 or higher.
+* [EDOT Collector](./edot-collector/index.md) in standalone mode.
+* [EDOT SDKs](./edot-sdks/index.md) instrumenting your application.
+
+The following versions of EDOT support central configuration:
+
+| Component | Minimum version |
+|-----------|----------------|
+| EDOT Collector | 8.19, 9.1 or higher |
+| EDOT Java | 1.5.0 or higher |
+| EDOT Node.js | 1.2.0 or higher |
+| EDOT PHP | 1.1.0 or higher |
+| EDOT Python | 1.4.0 or higher |
+
+::::{note}
+Serverless deployments are not currently supported.
+::::
+
+## Activate central management
+
+To activate central management for EDOT SDKs, follow these steps.
+
+:::::{stepper}
+
+::::{step} Edit the EDOT Collector configuration
+
+Edit the EDOT Collector configuration file to use central configuration. Refer to [EDOT Collector configuration](./edot-collector/config/default-config-standalone.md#central-configuration).
+
+Restart the Elastic Agent to apply the changes.
+
+::::
+
+::::{step} Edit the EDOT SDK configuration
+
+Activate the OpAMP client by setting the `ELASTIC_OTEL_OPAMP_ENDPOINT` environment variable to the URL endpoint that was configured in the EDOT collector in the previous step. Refer to each EDOT SDK documentation for more information:
```

Comment:
Is it written somewhere how to construct that endpoint?

Reply:
The EDOT SDK agent should be auto-appending that, it's better to not require that - fewer things to get wrong